### DISCUSSION TITLE
Team DIPY Citations

### DISCUSSION THREAD

**Discussion by ShreyasFadnavis** [dipy/dipy#3567] [1. Ideas & Feedback]
Hi Everyone,

I know that we do have a publications page for DIPY here: https://dipy.org/documentation/1.4.0./cite/
This page for some reason has not been updated and needs to be revamped. 

I also had a suggestion to segregate the publications into 2 sections:

- Research that comes from DIPY developers
- Research that is a re-implementation of other methods and now incorporated in DIPY

I was asked on multiple accounts by different users of DIPY for this and felt it might be a good idea to get this started. Again this is open to discussion -- Just put out an idea here!

We perhaps might also want to move this to a more prominent location of the software, right now its kinda hidden.

**Comment by arokem** [comment]
I think this is related to #1725, which would also be good to implement. If we point people to Zenodo as the authoritative citation, we can more nimbly add contributors on a release-by-release basis.
